README incorrectly instructs to run `dotnet build` from repository root

Description:
The repository README currently says to run `dotnet build` from the repository root to build everything. This is incorrect because there is no solution or project file at the repository root. Running `dotnet build` from the repository root produces an MSBuild error.

Steps to reproduce:
1. Open a terminal at the repository root.
2. Run: `dotnet build`

Observed result:
```
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

Expected result:
The README should either provide a command that points at an existing project or solution or instruct the user to run `dotnet build` from within a project folder. For example:
- `dotnet build ProjectEuler/ProjectEuler.csproj`

or

- Add a top-level solution file and instruct users to run `dotnet build` from the repository root.

Suggested fix:
Update `README.md` to replace the instruction "From the repository root build everything: `dotnet build`" with a command that targets an existing project (for example: `dotnet build ProjectEuler/ProjectEuler.csproj`) and/or mention that users can run `dotnet build` inside individual project folders.

Environment:
- PowerShell
- .NET SDK: .NET 10

Please label this issue as `documentation` and `bug`.

--- a/README.md
+++ b/README.md
@@ -1,12 +1,12 @@
 # Programming Exercises
 
-A collection of small programming exercise projects and challenges implemented in C# targeting .NET 10. Each subfolder contains an individual project and often a project-specific README with more details.
+A collection of small C# projects and challenges implemented in C# targeting .NET 10. Each subfolder contains an individual project and often a project-specific README with more details.
 
 [![Build & Tests](https://github.com/zhollis21/ProgrammingExercises/actions/workflows/BuildAndTestProjectEuler.yml/badge.svg)](https://github.com/zhollis21/ProgrammingExercises/actions) [![.NET](https://img.shields.io/badge/.NET-10-brightgreen)](https://dotnet.microsoft.com/) [![License](https://img.shields.io/github/license/zhollis21/ProgrammingExercises)](LICENSE)
 
 ## Overview
 
-This repository hosts small C# projects used for learning and practicing algorithms and problem solving. The most complete project is `ProjectEuler`, which contains a console application and unit tests for various Project Euler problems.
+This repository hosts a collection of small C# projects used for learning and practicing algorithms and problem solving. Each project lives in its own subfolder and includes a project-specific README with build and run instructions.
 
 ## Projects
 
@@ -17,13 +17,7 @@ This repository hosts small C# projects used for learning and practicing algorit
 ## Quick start
 
 1. Install the .NET 10 SDK: https://dotnet.microsoft.com/
-2. From the repository root build everything:
-
-`dotnet build`
-
-3. Run the Project Euler console app:
-
-`dotnet run --project ProjectEuler/ProjectEuler.csproj`
+2. See the README inside each project folder for build, run, and test instructions specific to that project.
 
 ## License
 

